reference/customers.v3.yml

```diff
@@ -986,57 +986,57 @@ paths:
           type: number
           name: limit
           description: Items count per page. `limit=50`
-        - name: 'name:in'
+        - name: 'name'
           in: query
           required: false
           type: string
-          description: Filter items by first_name and last_name.
+          description: Filter items by customer attribute name `name=Date+Joined`
         - name: 'name:like'
           in: query
           required: false
           type: string
-          description: Filter items by substring in first_name and last_name (%substring% - use with % symbols).
+          description: Filter items by substring in customer attribute name. `name:like=join`
```

Comment:
```suggestion
          description: Filter items by substring in customer attribute `name`. `name:like=doe`
```

Reply:
Similar to the above `name` query, `name:like=join` would return the custom `Date Joined` attribute I created. Maybe we need more info here?

Reply:
Looks good!